diff --git a/dist/index.mjs b/dist/index.mjs
index c2fad996791065001a450f710b1d75bac8e07311..5d62470c228a6cc0315605f0de5a0ff0f749a54c 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -135,12 +135,14 @@ function convertToOpenAICompatibleChatMessages(prompt) {
 function getResponseMetadata({
   id,
   model,
-  created
+  created,
+  citations,
 }) {
   return {
     id: id != null ? id : void 0,
     modelId: model != null ? model : void 0,
-    timestamp: created != null ? new Date(created * 1e3) : void 0
+    timestamp: created != null ? new Date(created * 1e3) : void 0,
+    citations
   };
 }
 
@@ -522,7 +524,8 @@ var OpenAICompatibleChatLanguageModel = class {
             if (delta.content != null) {
               controller.enqueue({
                 type: "text-delta",
-                textDelta: delta.content
+                textDelta: delta.content,
+                citations: value.citations
               });
             }
             if (delta.tool_calls != null) {
@@ -632,6 +635,7 @@ var OpenAICompatibleChatResponseSchema = z2.object({
   id: z2.string().nullish(),
   created: z2.number().nullish(),
   model: z2.string().nullish(),
+  citations: z2.array(z2.string()).nullish(),
   choices: z2.array(
     z2.object({
       message: z2.object({
@@ -661,6 +665,7 @@ var createOpenAICompatibleChatChunkSchema = (errorSchema) => z2.union([
     id: z2.string().nullish(),
     created: z2.number().nullish(),
     model: z2.string().nullish(),
+    citations: z2.array(z2.string()).nullish(),
     choices: z2.array(
       z2.object({
         delta: z2.object({
